Resources/Prototypes/DeltaV/Entities/Structures/Storage/Crates/barrel.yml

```diff
@@ -0,0 +1,155 @@
+    sprite: DeltaV/Objects/Storage/barrel.rsi
+    layers:
+    - state: base
+      map: ["enum.StorageVisualLayers.Base"]
+    - state: closed
+      map: ["enum.StorageVisualLayers.Door"]
+  - type: Icon
+    sprite: DeltaV/Objects/Storage/barrel.rsi
+    state: base
+  - type: Damageable
+    damageContainer: StructuralInorganic
+    damageModifierSet: Wood
+  - type: Destructible
+    thresholds:
+    - trigger:
+        !type:DamageTrigger
+        damage: 30
+      behaviors:
+      - !type:PlaySoundBehavior
+        sound:
+          collection: WoodDestroy
+      - !type:SpawnEntitiesBehavior
+        spawn:
+          MaterialWoodPlank1:
+            min: 1
+            max: 4
+      - !type:DoActsBehavior
+        acts: [ "Destruction" ]
+  - type: Construction
+    graph: WoodenBarrel
+    node: woodenbarrel
+    containers:
+    - entity_storage
+  - type: StaticPrice
+    price: 50
+  - type: Fixtures
+    fixtures:
+      fix1:
+        shape:
+          !type:PhysShapeAabb
+          bounds: "-0.2,-0.3,0.2,0.4"
+        density: 150
+        mask:
+        - SmallMobMask #this is so they can go under plastic flaps
+        layer:
+        - MachineLayer
+  - type: Climbable
+
+- type: entity
+  parent: StorageTank
+  id: WoodenKeg
+  name: wooden keg
+  description: A musty old wooden keg, with a tap attached to the front.
+  components:
+  - type: SolutionContainerManager
+    solutions:
+      tank:
+        maxVol: 500
+        canMix: true
+  - type: ExaminableSolution
+    solution: tank
+  - type: UserInterface
+    interfaces:
+      enum.TransferAmountUiKey.Key:
+        type: TransferAmountBoundUserInterface
+  - type: DrawableSolution
+    solution: tank
+  - type: InjectableSolution
+    solution: tank
+  - type: Drink
+    solution: tank
+  - type: Spillable
+    solution: tank
+  - type: DumpableSolution
+    solution: tank
+  - type: Sprite
+    sprite: DeltaV/Objects/Storage/keg.rsi
+    layers:
+    - state: base
+      map: ["enum.StorageVisualLayers.Base"]
+  - type: Damageable
+    damageContainer: StructuralInorganic
+    damageModifierSet: Wood
+  - type: StaticPrice
+    price: 50
+  - type: Construction
+    graph: WoodenKeg
+    node: woodenkeg
+    containers:
+    - entity_storage
+  - type: Fixtures
+    fixtures:
+      fix1:
+        shape:
+          !type:PhysShapeAabb
+          bounds: "-0.2,-0.3,0.2,0.4"
+        density: 200
+        mask:
+        - SmallMobMask #this is so they can go under plastic flaps
+        layer:
+        - MachineLayer
+  - type: Transform
+    noRot: false
+
+- type: entity
+  parent: WoodenKeg
+  name: root beer keg
+  description: A musty old wooden keg, with a tap attached to the front. Smells like root beer.
+  id: WoodenKegRootBeer
+  components:
+    - type: Label
+      currentLabel: reagent-name-root-beer
+    - type: SolutionContainerManager
+      solutions:
+        tank:
+          reagents:
+            - ReagentId: RootBeer
+              Quantity: 500
```

Comment:
for these unindent the components and reagent so - is inline with the field

Reply:
i think i did this right now... 👍